### PR TITLE
Feature/sticky note animation

### PR DESCRIPTION
**Pull request by tyleryy**
### Overview

Animation for resource sticky notes to fly in from the 4th wall (your face/screen) onto the page. The animation is triggered when the user scrolls down and the sticky note is in view.

### Details

Created a "slam in" animation variant that animates scale and opacity to achieve the effect. Converted the Resource Card component elements to motion elements to get the animation to "stagger" for each part of the sticky note.

### Addendum
Open to discussion on the timing and fine-grained details of the animation.

### CODE REVIEW

**Comment by tyleryy** [comment]
This PR can also be used for the mentor apply sticky notes.

**Comment by tyleryy** [comment]
I can also put the animation on the note cards and the stagger animation will be on the card and text.

File: apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss [hunk 1]
```diff
@@ -13,7 +13,9 @@
 	width: 238px;
```
Comment (samderanova) [inline apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss:13]: Issue: Slightly unrelated, but the width of the card has been hardcoded to a certain value (`344px`), which on the smallest mobile view, causes the sides of all cards to be cut off.

**Review by samderanova — CHANGES_REQUESTED** [review]
This is awesome! Thank you for working on this. A few things I'd like to point out:

File: apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss (hunk 1 above)
Comment (samderanova) [reply apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss:13]: I think we could just reduce the width to 300px

File: apps/site/src/app/(home)/sections/Landing/ApplyButton.tsx [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import Button from "react-bootstrap/Button";
 import StickerPosition from "@/components/Sticker/StickerPosition";
-import { HackSticker, HeartSticker } from "@/components/Sticker/Stickers";
+import {
+	PencilSticker,
+	HeartSticker,
+	HeadphoneSticker,
+	LaptopSticker,
+	MagnifyingGlassSticker,
+} from "@/components/Sticker/Stickers";
```
[the remark this answers is not in the document]
Comment (tyleryy) [reply apps/site/src/app/(home)/sections/Landing/ApplyButton.tsx:9]: shoot I think I branched incorrectly, I will fix that rn

**Review by samderanova — APPROVED** [review]
Great work on the animations!

**Pull request merged**